kratos/containers/pointer_vector_set.h

```diff
@@ -89,11 +89,13 @@ class PointerVectorSet final
     using const_reference = const TDataType&;
     using ContainerType = TContainerType;
 
+    /// Define the iterators
     using iterator = boost::indirect_iterator<typename TContainerType::iterator>;
     using const_iterator = boost::indirect_iterator<typename TContainerType::const_iterator>;
     using reverse_iterator = boost::indirect_iterator<typename TContainerType::reverse_iterator>;
     using const_reverse_iterator = boost::indirect_iterator<typename TContainerType::const_reverse_iterator>;
 
+    /// Other definitions
```

Comment:
same as above

Reply:
guys, no itention to add unnecessary noise, but i want to CC @sunethwarna since i know he is preparing modifications on this class (essentially about taking out the sorted part as i recall).

Reply:
I have already added do in aprevious PR, if there is conflict probably it is already conflicted